mne/epochs.py

```diff
@@ -272,9 +272,10 @@ def __init__(self, info, data, events, event_id, tmin, tmax,
             self.preload = False
             self._data = None
         else:
+            index_offset = 1 if data.shape[-1] > 1 else 0
             assert decim == 1
             if data.ndim != 3 or data.shape[2] != \
-                    round((tmax - tmin) * self.info['sfreq']) + 1:
+                    round((tmax - tmin) * self.info['sfreq']) + index_offset:
```

Comment:
Can you explain why this works? It seems like it shouldn't. This is meant to deal with the case where `tmax=tmin` and there is one sample of data, right?